### PR TITLE
Updated Dockerfile.dpdk1911 to use Centos8 and DPDK19.11.4

### PR DESCRIPTION
**Pull request by emmakenny**
For https://github.com/alauda/kube-ovn/issues/428

### CODE REVIEW

**Review by garyloug — CHANGES_REQUESTED** [review]
Hi @emmakenny , I'm happy with all changes, just some whitespace on the last line.

File: dist/images/Dockerfile.dpdk1911 [hunk 1]
```diff
@@ -15,21 +13,14 @@ ENV OVN_DIR=/usr/src/ovn
...
+RUN dnf install -y --enablerepo=PowerTools \
+  gcc make numactl-devel dpdk-devel meson \
+  unbound nc iptables ipset hostname && \
 # Install DPDK
   cd /usr/src/ && \
   curl http://fast.dpdk.org/rel/dpdk-${DPDK_VERSION}.tar.gz | tar xz && \
-  cd ${DPDK_DIR} && \
+  cd dpdk-stable-${DPDK_VERSION} && \
   meson builddir && cd builddir && \
-  meson configure -Dapps='pdump proc-info' -Dexamples='' -Dtests=false -Denable_kmods=false -Denable_docs=false && \
```
Comment (garyloug) [inline dist/images/Dockerfile.dpdk1911:32]: @emmakenny , we'll need to add this line back in. My fault this got dropped when moving to centos base.

**Review by garyloug — CHANGES_REQUESTED** [review]

File: dist/images/Dockerfile.dpdk1911 [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-# This package is required to build the OVS-DPDK RPMs and is available in Fedora 32.
-FROM fedora:32 as dpdk
+FROM centos:8 as dpdk
 ENV BUILD_DATE 20200731
 ENV RPM_ARCH=x86_64
 
-ENV DPDK_VERSION=19.11.1
-ENV OVS_VERSION=2.13.0
+ENV DPDK_VERSION=19.11.2
+ENV OVS_VERSION=2.13
```
Comment (oilbeater) [inline dist/images/Dockerfile.dpdk1911:6]: Can we upgrade to ovs 2.14 here?

Comment (oilbeater) [reply dist/images/Dockerfile.dpdk1911:6]: just notice the pr titile, should we use dpdk 19.11.4 here？

Comment (emmakenny) [reply dist/images/Dockerfile.dpdk1911:6]: Yes - sorry mistake, making that change as well as the other changes mentioned above now

File: dist/images/Dockerfile.dpdk1911 [hunk 3]
```diff
@@ -73,6 +65,6 @@ RUN dnf install -y kmod && dnf clean all
 COPY --from=rpm-builder /rpms/* /rpms/
 COPY start-ovs-dpdk.sh ovs-dpdk-healthcheck.sh uninstall.sh /kube-ovn/
 
-RUN  rpm -ivh --nodeps /rpms/*.rpm && \
+RUN rpm -ivh --nodeps /rpms/*.rpm && \
   rm -rf ${DPDK_DIR} /rpms && \
-  unset DPDK_DIR OVS_DIR OVN_DIR
+  unset DPDK_DIR OVS_DIR OVN_DIR
```
[the remark this answers is not in the document]
Comment (emmakenny) [reply dist/images/Dockerfile.dpdk1911]: Done

File: dist/images/Dockerfile.dpdk1911 (hunk 1 above)
Comment (emmakenny) [reply dist/images/Dockerfile.dpdk1911:32]: Done

File: dist/images/Dockerfile.dpdk1911 (hunk 2 above)
Comment (emmakenny) [reply dist/images/Dockerfile.dpdk1911]: Done

**Comment by garyloug** [comment]
Latest changes look good. Will build and test tomorrow.

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
@garyloug @emmakenny the commit looks good to me.Ping me if your test passed and I will merge it to master

**Comment by garyloug** [comment]
@oilbeater built and tested on my end, all good! Thanks @emmakenny

**Comment by oilbeater** [comment]
Thanks! @emmakenny @garyloug

**Pull request merged**